sql/binlog.cc

```diff
@@ -6790,6 +6794,167 @@ int MYSQL_BIN_LOG::purge_index_entry(THD *thd, ulonglong *decrease_log_space,
   DBUG_RETURN(error);
 }
 
+/**
+  Count a total size of binary logs (except the active one) to the variable
+  binlog_space_total.
+
+  @param need_lock_index  If true, this function acquires LOCK_index;
+                          otherwise the caller should already have acquired it.
+
+  @retval
+    0				ok
+  @retval
+    LOG_INFO_PURGE_NO_ROTATE	Binary file that can't be rotated
```

Comment:
LOG_INFO_PURGE_NO_ROTATE does not look like a value that could be returned by this function. Looking at the code, it's either 0, LOG_INFO_FATAL, or one of MYSQL_BIN_LOG::find_log_pos return values (LOG_INFO_IO, LOG_INFO_EOF)